ipfs/src/route.rs

```diff
@@ -119,7 +119,8 @@ mod tests {
 	use ethcore::client::TestBlockChainClient;
 
 	fn get_mocked_handler() -> IpfsHandler {
-		IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
+		//IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
```

Comment:
I think my preference would be to leave the original code, uncommented. Just keep the code as similar as possible but `--exclude` it. It doesn't matter really though, whatever you would prefer.